book.toml

```diff
@@ -12,15 +12,15 @@ edition = "2021"
 extra-watch-dirs = ["po"]
 
 [preprocessor.links]
-renderers = ["html"]
 
 [preprocessor.gettext]
+after = ["links"]
 
 [preprocessor.index]
-renderers = ["html"]
 
 [preprocessor.svgbob]
 renderers = ["html"]
+after = ["gettext"]
```

Comment:
Did you not say recently that the order is determined by the order in this file? :slightly_smiling_face:

Reply:
Ah, I see your commit message now!